content/manuals/admin/convert-account.md

```diff
@@ -45,6 +45,8 @@ Consider the following effects of converting your account:
 
 - The user account that you add as the first owner will have full administrative access to configure and manage the organization.
 
+- Converting a user account to an organization will delete all of the user's personal access tokens. See [Create an access token](https://docs.docker.com/security/for-developers/access-tokens/#create-an-access-token) for steps on creating personal access tokens after converting the user account.
```

Comment:
It looks good. For the link, I suggest changing it to a relative file link. If it's the full absolute URL like now, it shows a little icon next to the link that's intended for linking outside the docs. You may need to verify the suggested link change below because I usually use the `../security/for-developers/access-tokens.md/#create-an-access-token` syntax. I saw David use this syntax earlier, but I'm not 100% sure.

```suggestion
- Converting a user account to an organization will delete all of the user's personal access tokens. See [Create an access token](/manuals/security/for-developers/access-tokens.md/#create-an-access-token) for steps on creating personal access tokens after converting the user account.
```